### PR TITLE
Send email notification when a user signup or user update

### PR DESCRIPTION
**Pull request by RaghuMurugesan**
Fixes the issue #27

### CODE REVIEW

File: lib/mailer.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+			return console.log(error);
+		}
+		console.log('Mesage %s sent: %s', info.messageId, info.response);
+	});
+};
+
+function notifyAdmin(user) {
+	var rand;
+
+	var mailOptions = {
```
Comment (JMStudiosJoe) [inline lib/mailer.js]: my only comment is indenting is off should be 4 spaces imo.

File: lib/mailer.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+			'\t Phone : ' + user.phone + '\n' +
+			'\t Country : ' + user.country + '\n' +
+			'\t Region : ' + user.region + '\n' +
+			'\t Interest : ' + user.interests + '\n' +
+			'\t Skills : ' + user.skills + '\n' +
+			'\nRegards\n' +
+			'GiveLight'
+	};
+	sendEmail(mailOptions);
+	return rand;
```
Comment (JMStudiosJoe) [inline lib/mailer.js]: what is rand doing?

**Comment by RaghuMurugesan** [comment]
removed lines

File: lib/mailer.js (hunk 2 above)
Comment (RaghuMurugesan) [reply lib/mailer.js]: Oh, I forgot to remove it. Just updated

**Pull request merged**